metrics/influx/influx_test.go

```diff
@@ -31,6 +30,33 @@ func TestCounter(t *testing.T) {
 	}
 }
 
+func ExampleCounter() {
```

Comment:
Best not to mix Examples and real Tests in the same file. In the past I've stuck all the examples in an example_test.go file, can we do the same here?